tools/src/tokenize.rs

```diff
@@ -214,10 +214,7 @@ pub fn tokenize_yaml(string: &str) -> Vec<Token> {
             return true;
         }
 
-        match chars[cur_pos.get() - 2].1 {
-            ' ' | '\t' | '\r' | '\n' => true,
-            _ => false,
-        }
+        matches!(chars[cur_pos.get() - 2].1, ' ' | '\t' | '\r' | '\n')
```

Comment:
is this required?
to me, the previous one is easier to read, as it matches what other parts do for tokenization in general.

Reply:
This isn't really an “action” like the other match statements in the remainder of the function (it just evaluates to a boolean here), but I reverted that change anyway.

Note: just after writing this I noticed there's this function at the top of the file:
```
fn is_whitespace(ch: char) -> bool {
    ch == ' ' || ch == '\t' || ch == '\n' || ch == '\r'
}
```